unit-tests-jest/numbers.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * Returns a new array that contains only the even numbers in the given array.
+ */
+export function evenNumbers(numbers: number[]): number[] {
+  return numbers.filter((n) => n % 2 === 0);
+}
+
+/**
+ * Returns a number formatted in dollars and cents.
+ */
+export function toDollars(amount: number): string {
+  return `$${amount}.00`;
+}
+
+/**
+ * Returns a new array of numbers where every entry has been divided by the
+ * given divisor. Does not modify the original array.
+ */
+export function divideBy(numbers: number[], divisor: number): number[] {
+  for (let i = 0; i < numbers.length; i++) {
```

Comment:
This modifies the array, while the description says it returns a new array and does not modify the original array.